$OpenBSD$

Remove gtk-3.0 >= 3.6 check. Not only does it create
bogus flags that break compilation, our gtk+3 is 3.12+

--- configure.ac.orig	Tue Sep  2 06:41:54 2014
+++ configure.ac	Sun Sep 21 01:24:20 2014
@@ -71,7 +71,7 @@ LIBVANUBI_VALAFLAGS="$LIBVANUBI_VALAFLAGS --pkg posix 
 LIBVANUBI_CFLAGS="$LIBVANUBI_CFLAGS -w"
 LIBVANUBI_LIBS="$LIBVANUBI_LIBS"
 
-VALA_CHECK_MODULES(VANUBI, [gtk+-3.0 >= 3.6 gio-unix-2.0 gtksourceview-3.0 vte-2.90 gdk-x11-3.0 libwnck-3.0])
+VALA_CHECK_MODULES(VANUBI, [gtk+-3.0 gio-unix-2.0 gtksourceview-3.0 vte-2.90 gdk-x11-3.0 libwnck-3.0])
 dnl We ignore GTK_3_10_*
 PKG_CHECK_MODULES(GTK_3_10, [gtk+-3.0 >= 3.10],
 				  [VANUBI_VALAFLAGS="$VANUBI_VALAFLAGS -D GTK_3_10"],
